kustomize/commands/edit/add/flagsandargs.go

```diff
@@ -13,6 +13,15 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/filesys"
 )
 
+const (
```

Comment:
q: does this `flagsandargs` file only have things relevant to configmap/secret? If so, maybe it should be renamed to `configmapandsecretargs.go` to be more specific

Reply:
Yes, it's used solely on those two places at the moment. I'll rename the types and files to reflect that.

Reply:
Renamed & updated!